go/host/container/cli_flags.go

```diff
@@ -64,5 +66,7 @@ func getFlagUsageMap() map[string]string {
 		useInMemoryDBName:            "Whether the host will use an in-memory DB rather than persist data",
 		levelDBPathName:              "Filepath for the levelDB persistence dir (can be empty if a throwaway file in /tmp/ is acceptable or if using InMemory DB)",
 		debugNamespaceEnabledName:    "Whether the debug names is enabled",
+		batchIntervalName:            "How many seconds between each batch",
```

Comment:
If you want to use `ParseDuration` then should probably change these prompts to not mention "seconds", people can put any duration with units I guess.